lib/fs.js

```diff
@@ -411,16 +375,8 @@ fs.accessSync = function(path, mode) {
   }
 };
 
-// fs.exists never throws even when the arguments are invalid - if there is
-// a callback it would invoke it with false, otherwise it emits a warning
```

Comment:
If we throw on invalid callbacks in `fs.exists` now, it might make sense to throw on other invalid arguments as well...I don't feel strongly about fixing that in this PR though.

Reply:
I would rather fix that in another PR.